.github/workflows/consumer_contract_tests.yml

```diff
@@ -0,0 +1,154 @@
+name: Consumer contract tests
```

Comment:
This is largely a copy of https://github.com/DataBiosphere/leonardo/blob/develop/.github/workflows/consumer_contract_tests.yml with leo/sam renamed to rawls/BPM. We aren't trying to factor more of this into shared github actions/workflows because the mechanism is changing to Sherlock doing the version tracking.